app/views/subjects/show.html.erb

```diff
@@ -10,7 +10,7 @@
 
   <p>
     <strong>Participant:</strong>
-    <%= @subject.participant.try(:name) %>
+    <%= @subject.questioner.try(:name) %>
```

Comment:
Seems like you changed the relation's name (it's "questioner" now) but not the description (still "Participant"); which one is right ?
Having differences between code and concepts is dangerous for global understanding (how will a futur team member will understand this ?)